### PR TITLE
Add more vmware fusion binaries

### PR DESCRIPTION
**Pull request by n0ts**

### CODE REVIEW

File: Casks/vmware-fusion.rb [hunk 1]
```diff
@@ -10,11 +10,34 @@
...
+vmware-usbarbitrator
+vmware-vdiskmanager
+vmware-vmdkserver
+vmware-vmx
+vmware-vmx-debug
+vmware-vmx-stats
+  }.each do |file|
+    binary "VMware Fusion.app/Contents/Library/#{file}"
+  end
+  binary "VMware Fusion.app/Contents/Library/VMware OVF Tool/ovftool"
```
Comment (vitorgalvao) [inline Casks/vmware-fusion.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

File: Casks/vmware-fusion.rb [hunk 2]
```diff
@@ -10,11 +10,34 @@
   auto_updates true
 
   app 'VMware Fusion.app'
-  binary 'VMware Fusion.app/Contents/Library/vmnet-cfgcli'
-  binary 'VMware Fusion.app/Contents/Library/vmnet-cli'
-  binary 'VMware Fusion.app/Contents/Library/vmrun'
-  binary 'VMware Fusion.app/Contents/Library/vmware-vdiskmanager'
-  binary 'VMware Fusion.app/Contents/Library/VMware OVF Tool/ovftool'
+  %w{
+vmnet-bridge
```
Comment (vitorgalvao) [inline Casks/vmware-fusion.rb]: Please indent these.

**Comment by n0ts** [comment]
@vitorgalvao Thanks! I fixed indent.

**Comment by vitorgalvao** [comment]
You’re updating an unrelated package.

**Comment by n0ts** [comment]
@vitorgalvao I fix it.

**Comment by vitorgalvao** [comment]
Travis is complaining, as it’s breaking one of the style rules.

At least for now, it might be best to just repeat the `binary` stanza, like [terraform](https://github.com/caskroom/homebrew-cask/blob/master/Casks/terraform.rb).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 32fc83be2b20984eca20f108e5110efca638e32a. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**